src/pages/groupComparison/ClinicalDataEnrichmentsTable.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import * as _ from "lodash";
+import { observer } from "mobx-react";
+import autobind from 'autobind-decorator';
+import LazyMobXTable from 'shared/components/lazyMobXTable/LazyMobXTable';
+import { toConditionalPrecision } from 'shared/lib/NumberUtils';
+import { ClinicalDataEnrichmentStore } from './ClinicalData';
+import { ClinicalDataEnrichment } from 'shared/api/generated/CBioPortalAPIInternal';
+
+export interface IClinicalDataEnrichmentsTableProps {
+    dataStore: ClinicalDataEnrichmentStore;
+}
+
+export enum ClinicalDataEnrichmentTableColumnType {
```

Comment:
is this used?

Reply:
I think it should be, looks like the only way to do that is to make this a string-valued enum, let them take on the column names, and use them there too